internal/index/indexscanner/scanner_test.go

```diff
@@ -96,24 +96,58 @@ func TestReadPluginFile(t *testing.T) {
 }
 
 func TestReadReceiptFromFile(t *testing.T) {
-	tmpDir, cleanup := testutil.NewTempDir(t)
-	defer cleanup()
-
-	name := "foo"
-	wantStatus := &index.ReceiptStatus{
-		Source: index.SourceIndex{
-			Name: name,
+	type args struct {
```

Comment:
for this time it's ok but in the future I recommend avoiding single-field `type args`

Reply:
Crap sorry about that, you mentioned that before. I'll remember next time and change this in a follow up PR most likely.